test-data/unit/stubgen.test

```diff
@@ -81,20 +96,45 @@ def g(): ...
 [case testVariable]
 x = 1
 [out]
-x = ...  # type: int
+x: int
+
+[case testAnnotatedVariable]
+x: int = 1
+[out]
+x: int
+
+[case testAnnotatedVariableGeneric]
+x: Foo[int, str] = ...
+[out]
+x: Foo[int, str]
+
+[case testAnnotatedVariableOldSyntax]
+x = 1  # type: int
+[out]
+x: int
+
+[case testAnnotatedVariableNone]
+x: None
+[out]
+x: None
+
+[case testAnnotatedVariableNoneOldSyntax]
+x = None  # type: None
+[out]
+x: None
 
 [case testMultipleVariable]
 x = y = 1
 [out]
-x = ...  # type: int
-y = ...  # type: int
+x: int
+y: int
 
 [case testClassVariable]
 class C:
     x = 1
 [out]
 class C:
-    x = ...  # type: int
+    x: int
```

Comment:
For class attributes it is important to know whether an attribute was just _declared_ like ``x: int`` or actually defined like ``x: int = 1``. There is a difference for protocol classes -- in former case an explicit subclass cannot be instantiated. Therefore, I think the translation for class attributes should be like this:
```
x: int -> x: int
x = 1 -> x: int = ...
x: int = 1 -> x: int = ...
```

Reply:
OK... what about instance attributes that stubgen finds because of self accesses? (for example, when you have `self.x = 1` in `__init__`). Should those be just `x: int` or should they also be `x: int = ...`?

Reply:
> OK... what about instance attributes that stubgen finds because of self accesses? (for example, when you have ``self.x = 1`` in ``__init__``). Should those be just ``x: int`` or should they also be ``x: int = ...``?

I think it should be ``x: int = ...`` if there is an assignment in ``__init__`` (again mostly from the protocols POW).

Reply:
OK, I've made the change (and some others), I will be pushing un update after some testing

Reply:
Great!